### PR TITLE
Windows.cfg: add the cfg for case balloon_hotplug.

### PR DESCRIPTION
**Pull request by CongSmile**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1285979

### CODE REVIEW

**Comment by CongSmile** [comment]
Based on: https://github.com/autotest/tp-qemu/pull/519

**Comment by CongSmile** [comment]
Close this PR.

As using the Qdevice to implement balloon hotplug, instead of using case pci_hotplug.

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

File: shared/cfg/guest-os/Windows.cfg [hunk 1]
```diff
@@ -276,7 +276,7 @@
         analyze_cmd = E:\autoit3.exe E:\dump_control.au3
         sub_type = nmi_bsod_catch
         rename_error_image = yes
-    balloon_check:
+    balloon_check, balloon_hotplug:
         free_mem_cmd = wmic os get FreePhysicalMemory
```
Comment (suqinhuang) [inline shared/cfg/guest-os/Windows.cfg]: need double check the cmd

Comment (CongSmile) [inline shared/cfg/guest-os/Windows.cfg]: cmd = 'systeminfo | findstr /C:"Available Physical Memory"'

Comment (CongSmile) [inline shared/cfg/guest-os/Windows.cfg]: There is no problem for the cmd, as there is a ratio to handle the free mem difference for above 2 cases.

**Comment by xutian** [comment]
that's ok for me.

**Pull request merged**